ui/src/apps/diagnose/components/DiagnoseGenerator.tsx

```diff
@@ -52,18 +64,49 @@ export default function DiagnoseGenerator() {
         >
           <DatePicker showTime />
         </Form.Item>
-        <Form.Item
-          name="rangeDuration"
-          rules={[{ required: true }]}
-          label={t('diagnose.generate.range_duration')}
-        >
-          <Select style={{ width: 120 }}>
-            <Select.Option value={5}>5 min</Select.Option>
-            <Select.Option value={10}>10 min</Select.Option>
-            <Select.Option value={30}>30 min</Select.Option>
-            <Select.Option value={60}>1 hour</Select.Option>
-            <Select.Option value={24 * 60}>1 day</Select.Option>
-          </Select>
+        <Form.Item label={t('diagnose.generate.range_duration')}>
```

Comment:
We can `required` props so the asterisk won't miss.

Reply:
It doesn't work. The star does not appear. Maybe it is because there is no values inside.

Reply:
I mean to use `required` as props, likes `<Form.Item required .../>`, I tested and it works:

![image](https://user-images.githubusercontent.com/1284531/77819208-fe1f4180-7113-11ea-9afd-c657cd28c4a9.png)